app/commands/v2/publish.rb

```diff
@@ -116,9 +130,15 @@ def validate_update_type
       end
 
       def delete_change_notes
+        Rails.logger.debug("Deleting change notes")
         ChangeNote.where(edition:).delete_all
       end
 
+      def supersede_previous_edition
```

Comment:
do you want this to be called somewhere Kevin?

Reply:
Yes - great spot!